examples/add_openshift_provider.yml

```diff
@@ -19,5 +19,25 @@
       miq_verify_ssl: false
       provider_verify_ssl: false
     register: result
+  - debug: var=result
+
+  - name: Add Openshift Containers Provider with Prometheus endpoint to ManageIQ
+    manageiq_provider:
+      name: 'Openshift02'
+      provider_type: 'openshift-origin'
+      state: 'present'
+      provider_api_hostname: 'os02.example.com'
+      provider_api_port: '8443'
+      provider_api_auth_token: '******'
+      metrics: true
+      metrics_type: 'prometheus'
+      prometheus_hostname: 'prometheus01.example.com'
+      prometheus_port: '80'
```

Comment:
So how about:
metrics : 'hawkular' / 'prometheus' / none (by default)
Then of course we could have `metrics_port, metrics_hostname` regardless of the venue.

But this is not backwards compatible with old playbooks. do we want it to be?